tasks/strings/strings.js

```diff
@@ -4,23 +4,38 @@
  * @param {string} str
  */
 function reverseString(str) {
+  if (typeof (str) == 'number') return 'This is not a string!';
+  let arr = str.split('');
+  let arrReverse = arr.reverse();
+  let strReverse = arrReverse.join('');
+  return strReverse;
 }
 
 /**
  * 1. Calculate a century by given year
  * if a number is passed
-* @param {number} year
-*/
+ * @param {number} year
+ */
 function centuryFromYear(year) {
-
+  let str = String(year); // the logic for non 4 digits year is not realised
```

Comment:
it is toooooo complex :)
please use Math.ceil() here

Reply:
fixed